radio/src/targets/common/arm/stm32/telemetry_driver.cpp

```diff
@@ -45,17 +45,19 @@ static void telemetryInitDirPin()
   GPIO_InitStructure.GPIO_OType = GPIO_OType_PP;
   GPIO_InitStructure.GPIO_Pin   = TELEMETRY_DIR_GPIO_PIN;
   GPIO_Init(TELEMETRY_DIR_GPIO, &GPIO_InitStructure);
-  GPIO_ResetBits(TELEMETRY_DIR_GPIO, TELEMETRY_DIR_GPIO_PIN);
+  TELEMETRY_DIR_INPUT();
 }
 
-void telemetryPortInit(uint32_t baudrate, uint8_t mode)
+void telemetryPortInitCommon(uint32_t baudrate, uint8_t mode, uint8_t noinv = 0)
```

Comment:
In future, I would suggest camelCase for the variable names like noInv, and using a define to make the 0/1 parameter have meaning, as well as then being what is compared with in later tests... i.e., at line 90.